ui/app/templates/components/secret-edit.hbs

```diff
@@ -69,16 +69,11 @@
         @showAdvancedMode={{this.showAdvancedMode}}
         @modelForData={{this.modelForData}}
         @error={{this.error}}
-        @isV2={{this.isV2}}
-        @secretData={{this.secretData}}
         @buttonDisabled={{this.buttonDisabled}}
-        @canUpdateSecretMetadata={{this.canUpdateSecretMetadata}}
         @canReadSecretData={{this.canReadSecretData}}
-        @canReadSecretMetadata={{this.canReadSecretMetadata}}
```

Comment:
Should we remove `isV2`, `secretData`, `canUpdateSecretMetadata`, `canReadSecretMetadata` in the `secret-edit` component file too?

Reply:
I was just trying to break some of these tasks up into smaller ones by starting with the child components of this file - but yes! That will come!